charts/karmada-operator/values.yaml

```diff
@@ -80,3 +80,39 @@ operator:
     rollingUpdate:
       maxUnavailable: 0
       maxSurge: 50%
+
+## operator service config
+service:
+  ## @param service.name
+  name: metrics
+  ## @param service.type
+  type: ClusterIP
+  ## @param service.port port of metrics a port
+  port: 8080
+  annotations: {}
+
+serviceMonitor:
```

Comment:
Today, does the operator expose any metrics? I don't think it does. You're planning on adding some metrics for it?

Reply:
*_build_info 
ref: https://github.com/karmada-io/karmada/pull/6215